src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -20,13 +21,17 @@ namespace Microsoft.AspNet.Identity
     public class RoleManager<TRole> : IDisposable where TRole : class
     {
         private bool _disposed;
-        private HttpContext _context;
+        private readonly HttpContext _context;
 
         /// <summary>
-        /// Constructor
+        ///     Constructor
         /// </summary>
         /// <param name="store">The IRoleStore commits changes via the UpdateAsync/CreateAsync methods</param>
-        /// <param name="roleValidator"></param>
+        /// <param name="roleValidators">IEnumerable of role validators</param>
+        /// <param name="keyNormalizer">user property normalizers</param>
+        /// <param name="errors">IdentityErrorDescribers</param>
```

Comment:
OT: Parameter name should probably be errorDescriber unless it is the list of errors. Also, I noticed in the body of the constructor that we initialize a couple of classes if null. Is this the pattern we agreed to get rid of or something else?